### DISCUSSION TITLE
How to get the value of the Select component?

### DISCUSSION THREAD

**Discussion by KonradLinkowski** [JedWatson/react-select#5010] [Q&A] [answered]
Hi, I'm new to React and using custom react components.
**How can I get the value of the Select?**
I couldn't find any example in the docs.

**Comment by karlkras** [comment, marked as answer]
You get this out of the value of the event from your onChange handler:
```
<ReactSelect
...
   onChange={event => processChange(event)}
...
/>
```       

and your handler:
```
    const processChange = (event: React.ChangeEvent<HTMLSelectElement>) => {
        // event will have whatever was selected.
    }
```      

hope this helps.

**Reply by KonradLinkowski** [reply]
That helped a lot.
I just add that a state needs to be created to save a variable somewhere.
